apps/i18n/javalab/en_us.json

```diff
@@ -32,6 +32,7 @@
   "enablePeerReview": "Enable Peer Review",
   "errorJavabuilderConnectionGeneral": "We hit an error connecting to our server. Try again.",
   "errorJavabuilderConnectionNotAuthorized": "You are not authorized to access Javalab. Do you need to log in or join a section?",
+  "errorLoadingClassmates": "There was an error loading the list of your classmates. Switch to 'Instructions' and back to 'Review' to reload the list.",
```

Comment:
Just checking, this will change soon once we can refresh on opening the dropdown right?

Reply:
Yes